### DISCUSSION TITLE
Is this plugin compatible with Medusa 2.0 ?

### DISCUSSION THREAD

**Discussion by tonai** [adrien2p/medusa-plugins#185] [Q&A] [answered]
Is this plugin ([medusa-plugin-auth](https://github.com/adrien2p/medusa-plugins/tree/main/packages/medusa-plugin-auth)) compatible with Medusa 2.0 ?
Thanks

**Comment by adrien2p** [comment, marked as answer]
Hi @tonai no it is not. I might migrate it at some point or wait for the community to provide any interest towards it. But in medusa v2 you can create a provider for new auth method and you can have a look at the google one for example in medusa doc
